main.go

```diff
@@ -214,6 +217,9 @@ func init() {
 			if config.AWS.SNS.TopicArn != "" {
 				enabledOutputsText += "AWSSNS "
 			}
+			if config.AWS.CloudWatchLogs.LogGroup != "" {
+				enabledOutputsText += "AWSCLOUDWATCHLOGS "
```

Comment:
How about we change the output text like below:

```suggestion
				enabledOutputsText += "AWSCloudWatchLogs "
```

We use camel case for other products(AWSLambda, Cloud PubSub, etc...)

Reply:
i dont know why I insist in all caps 😂